test/CommandLine.Test/ListCommandTests.cs

```diff
@@ -180,7 +180,7 @@ public void GetPackageReturnsAllVersionsIfAllVersionsFlagIsSet()
 
             // Assert
             Assert.Equal(5, packages.Count());
-            AssertPackage(new { Id = "jQuery", Ver = "1.44" }, packages.ElementAt(0));
+            AssertPackage(new { Id = "JQuery", Ver = "1.44" }, packages.ElementAt(0));

```

Comment:
This should be a separate test, each test should represent one feature/behavior tested to the degree possible